### PR TITLE
tests: simplify expected output

### PR DESCRIPTION
**Pull request by giuseppe**
the condition doesn't work when the runtime to use is specified through its absolute path as the error message contains that.

Simplify the check and just look for "read from the init process".

Hitting this issue in the crun CI

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: test/e2e/run_seccomp_test.go [hunk 1]
```diff
@@ -39,10 +39,7 @@ var _ = Describe("Podman run", func() {
 			//   127 + failed to connect to container's attach socket ... ENOENT
 			Expect(session.ExitCode()).To(BeNumerically(">=", 126), "Exit status using runc")
 		} else {
-			expect := "OCI runtime error: crun: read from the init process"
-			if IsRemote() {
-				expect = "for attach: crun: read from the init process: OCI runtime error"
-			}
+			expect := "read from the init process"
```
Comment (edsantiago) [inline test/e2e/run_seccomp_test.go:42]: I really dislike loose error-message checks, so I won't be approving this. I won't block, either.

However, please get rid of the temporary variable, it's no longer necessary.

Comment (giuseppe) [reply test/e2e/run_seccomp_test.go]: reverted to use the full message with `podmanTest.OCIRuntime`.

And added another fix on top of it when the OCI runtime is neither crun nor runc

**Comment by edsantiago** [comment]
/lgtm

Very nice, thank you!

**Comment by mheon** [comment]
/hold
gofmt is angry

**Comment by giuseppe** [comment]
> /hold gofmt is angry

thanks, fixed now

**Pull request merged**